app/assets/javascripts/account/templates/account-settings.html.slim

```diff
@@ -105,20 +105,66 @@
                 .form-group-month.large-2.columns
                   label.sr-only for="date_of_birth_month"
                     | {{'label.dob_month' | translate}}
-                  input(ng-model="userAuth.contact.dob_month" ng-required="true" maxlength="2" name="date_of_birth_month" placeholder="MM" type="text" ng-class="{ error: inputInvalid('date_of_birth_month', '', form.accountNameDOB) }" ng-disabled="isLocked('dob')" pattern="\d*" ui-validate="'DOBValid(\'month\', $value)'" numeric-only="" autocomplete="off" ng-change="recheckDOBDay('accountNameDOB'); displayChangeNotice('nameOrDOBChanged')" aria-describedby="date_of_birth_error")
+                  input(
```

Comment:
reformatted to make this easy to read, but makes the diff hard to read. Keys I touched throughout were `ng-change` and `ui-validate`